scripts/teamcity.sh

```diff
@@ -23,6 +23,10 @@ cp -r frontend/build/* backend/public
 # On Teamcity the JDeb build doesn't seem to follow the symbolic link while packaging, weirdly
 cp frontend/node_modules/pdfjs-dist/build/pdf.worker.min.js backend/public/third-party/pdf.worker.min.js
 
+#Use java 11
+export JAVA_HOME=/usr/lib/jvm/java-11-amazon-corretto
```

Comment:
I'm guessing this is implicitly defined by the AMI we expect Giant to be running on?